api/operator/v1alpha1/authentication_types.go

```diff
@@ -139,6 +139,8 @@ type ConfigSpec struct {
 	ICPPort                     int32          `json:"icpPort"`
 	FIPSEnabled                 bool           `json:"fipsEnabled"`
 	ROKSEnabled                 bool           `json:"roksEnabled"`
+	AuditUrl                    string         `json:"auditUrl,omitempty"`
```

Comment:
If these aren't required, they should be `*string`, and `nil` checks should happen on accesses.